src/Sarif/PropertyBag.cs

```diff
@@ -16,7 +16,12 @@ public class PropertyBagDictionary : TypedPropertyBagDictionary<object>
     {
         internal const string DEFAULT_POLICY_NAME = "default";
 
-        public PropertyBagDictionary() : base() { }
+        public PropertyBagDictionary() : this(null) { }
+
+        public PropertyBagDictionary(PropertyBagDictionary initializer) :
+            this(initializer, null)
+        {
+        }
 
         public PropertyBagDictionary(
             PropertyBagDictionary initializer = null,
```

Comment:
Remove the argument initializer ("`= null`"). Remember that that's why we added the overloads: to ensure that all languages could have the convenience of the abbreviated argument lists (not all languages understand default parameters).